index.bs

```diff
@@ -101,21 +118,190 @@ spec: webidl
 </section>
 <section>
   <h2 id="permission-descriptor">
-    Permission descriptor
+    Descriptions of permission requests
   </h2>
   <pre class='idl' title=''>
     dictionary PermissionDescriptor {
       required PermissionName name;
     };
   </pre>
   <p>
-    A permission is described by a name and other properties that depend on the
-    name. The simplest permissions require only a name, but some others have
-    more detailed structure that requires more information to describe it. In
-    that case, they should define a customized <a>permission descriptor
-    type</a> dictionary that inherits from {{PermissionDescriptor}}.
+    Each powerful feature has one or more aspects that websites may request
```

Comment:
I don't think there is a definition of "powerful features". It used to be in Secure Contexts, but it got taken out. I was going back and forth on adding a "Definitions" section in this patch, but I guess I'll do it for this one.

And I've switched to "can".